### ISSUE TITLE
Create Hyper-V Gen2 image for download

### ISSUE THREAD

**Issue by WSLUser** [clearlinux/distribution#2242]
Gen 2 for Hyper-V has been out for years now and increases performance drastically. See for differences: https://www.nakivo.com/blog/hyper-v-generation-1-vs-2/. There are steps to create a Gen2 VM from a Gen1 but it's much less user friendly for Linux VMs. It would be better instead for Clear Linux to simply offer a Gen2 image (vhdx) for download at https://clearlinux.org/downloads. I'm pretty sure even Azure is using Gen2 at this point for most distros (Actually Azure is upstream for Hyper-V changes).

**Comment by sala91** [comment]
Agreed. Would be great for trying out before deciding to ditch Windows.

**Comment by WSLUser** [comment]
I would check out WSL2 before ditching Windows. Just import the docker image and add a user. I just want another option for cases in which a full VM may be needed.

**Comment by WSLUser** [comment]
For Azure, I think a better ask would be to be placed on the Azure Marketplace if they aren't already there but they'll still need to bump their VM build to Gen 2 anyways. If they at least made their build process more transparent for how they originally did Gen2, that would be very helpful.  I did try the old Gen2 image before but appears to of been corrupted as I couldn't use it for WSL2 or Hyper-V. Makes me think some changes were made to the Gen2 VM generation, which isn't surprising.

**Comment by ThaDaVos** [comment]
Any update on this? Pretty sad a Gen2 image isn't provided anymore... at least no up to date - currently trying the one from https://cdn.download.clearlinux.org/releases/32980/clear/

**Comment by ephut** [comment]
The tooling was in pretty bad shape so we stopped, but we'll check again.

**Comment by ThaDaVos** [comment]
I also notice that the kernel used with `kernel-hyperv` is pretty old:
`5.6.7-285.hyperv`

I am getting some errors when the image boots and found only that it may be because of an outdated kernel:
![afbeelding](https://github.com/user-attachments/assets/c2cbdc9e-42cc-4dec-9a52-6587a9f7bbd2)

**Comment by ThaDaVos** [comment]
It seems the Hyper-V kernel is just outdated:
![afbeelding](https://github.com/user-attachments/assets/49cf8ce0-4e20-4c9a-84bc-10686124835e)
